src/main/java/htsjdk/tribble/gff/Gff3Codec.java

```diff
@@ -190,15 +191,15 @@ private Gff3Feature decode(final LineIterator lineIterator, final DecodeDepth de
      * @return map of keys to values for attributes of this feature
      * @throws UnsupportedEncodingException
      */
-    static private Map<String,String> parseAttributes(final String attributesString) throws UnsupportedEncodingException {
-        final Map<String, String> attributes = new LinkedHashMap<>();
+    static private Map<String, List<String>> parseAttributes(final String attributesString) throws UnsupportedEncodingException {
+        final Map<String, List<String>> attributes = new LinkedHashMap<>();
         final List<String> splitLine = ParsingUtils.split(attributesString,ATTRIBUTE_DELIMITER);
         for(String attribute : splitLine) {
             final List<String> key_value = ParsingUtils.split(attribute,KEY_VALUE_SEPARATOR);
             if (key_value.size()<2) {
```

Comment:
That does seem reasonable.  Honestly, I can't see why failure of this check leads to a continue as opposed to throwing an exception, since this indicates a invalid file.

Reply:
Lets make an exception then..